docs/api/session.md

```diff
@@ -1330,18 +1330,35 @@ the initial state will be `interrupted`. The download will start only when the
 
 Returns `Promise<void>` - resolves when the session’s HTTP authentication cache has been cleared.
 
-#### `ses.setPreloads(preloads)`
+#### `ses.setPreloads(preloads)` _Deprecated_
```

Comment:
We could mention here that people should use `registerPreloadScript` instead.

Reply:
I may need to collapse some commits. This was added in 5f90993552b1657bd3d2f175dc3f0fc640e7bd3a

Reply:
I've now collapsed all of the feedback into the original 6 commits. Sorry about the confusion.